### DISCUSSION TITLE
[DEPRECATING] removal of k3sCollection type from codebase

### DISCUSSION THREAD

**Discussion by dweomer** [hauler-dev/hauler#357] [Deprecations]
As per the below PR(s), support for the `k3sCollection` type will be removed quite soon (currently targeted for [1.1-next1](https://github.com/hauler-dev/hauler/milestone/11))
- https://github.com/hauler-dev/hauler/pull/352

If you find yourself using this please reach out and share some deets.

We will quite likely be replacing this functionality with something similar in 2025!

**Comment by zackbradys** [comment]
Closing as resolved due to https://github.com/hauler-dev/hauler/releases/tag/v1.1.1
